requirements.txt

```diff
@@ -3,7 +3,7 @@ httplib2==0.20.4; python_version > '3'
 pysimplesoap==1.08.14; python_version <= '2.7'
 git+https://github.com/pysimplesoap/pysimplesoap.git@py311#pysimplesoap;
 cryptography==3.3.2; python_version <= '2.7'
-cryptography==3.4.7; python_version > '3'
+cryptography==41.0.0; python_version > '3'
```

Comment:
Hi @HanslettTheDev!. Check the cryptography version for python 3. There is no 41 yet. The last in pypi is 40.0.2 and @salticus in his update uses 39.0.2 version.

Reply:
Thanks, I think i will fix that as well. I'm still making some changes